spec/services/analytics_spec.rb

```diff
@@ -358,4 +358,82 @@
       end
     end
   end
+
+  context 'with an SP request_url saved in the session' do
+    context 'no request_url' do
+      let(:session) { { sp: { acr_values: Saml::Idp::Constants::IAL1_AUTHN_CONTEXT_CLASSREF } } }
+
+      let(:expected_attributes) do
+        {
+          sp_request: {
+            component_values: { 'ial/1' => true },
+            component_separator: ' ',
```

Comment:
unrelated to this specific PR, `component_separator` seems like a silly thing to include in analytics to me? I guess if we deprecate VOTs it's going away anyways.